projects/igniteui-angular/src/lib/icon/icon.service.ts

```diff
@@ -164,7 +161,7 @@ export class IgxIconService {
      */
     public isSvgIconCached(name: string, family: string = ''): boolean {
         const familyClassName = this.familyClassName(family);
-        if (this._cachedSvgIcons.has(familyClassName)) {
+        if (this._cachedSvgIcons.has(familyClassName) && this._platformUtil.isBrowser) {
```

Comment:
Remove this check. In platform server the check would just return `false`, so nothing will throw. If we implement server handling in the future, then the method will return the icon properly on the server also.